assets/src/exit-modal-survey/index.js

```diff
@@ -8,7 +8,7 @@ import ExitModal, { CLOSE_MODAL_EVENT } from '@eventespresso/react-exit-modal-ty
 import './style.css';
 import { data } from '@eventespresso/eejs';
 import { stringify } from 'querystringify';
-import { __ } from '@wordpress/i18n';
+import { __ } from '@eventespresso/i18n';
```

Comment:
why would we do this if this functionality is now located at `eejs.i18n` ?

Reply:
`@eventespresso/i18n` is an alias (via webpack) for `eejs.i18n`.  This gives us flexibility in case in the future we need to point `@eventespresso/i18n` to something else.  See [this change](https://github.com/eventespresso/event-espresso-core/pull/393/commits/d7ad7d98e23564604634cf156da34c36a951d026#diff-5c572bb8f24a00a84aa28f22d3a39ba5R9)